apidoc/Titanium/Geolocation/Geolocation.yml

```diff
@@ -85,7 +85,7 @@ description: |
     "When in Use" to "Always", which is the recommended way for managing location permissions in iOS 11 and later.
     Please also remember to request your desired location-permissions before using any geolocation-related API in 
     order to receive the best usability and permission-control during the app-lifecycle using <Titanium.Geolocation.hasLocationPermissions>
-    and <Ti.Geolocation.requestLocationPermissions>. Also note that you also need to include the `NSLocationWhenInUseUsageDescription` key 
+    and <Titanium.Geolocation.requestLocationPermissions>. Also note that you also need to include the `NSLocationWhenInUseUsageDescription` key 
```

Comment:
This is a real pain when not knowing how the docs generator works. Can't we just allow both and move the general usage to "Ti"? Who uses "Titanium.UI.createWindow" ?

Reply:
We should be able to do that. From my understanding of all this, you reference by the `name` property in the yaml file that defines that API. 

So, in theory we should be able to replace all of these with `Ti.X` and be done with it. But I imagine we'd probably run into a load of issues regarding, links going dead, search results being bad etc. A better option may just be to rewrite the text to `Ti.X` when we write the result.